skymp5-client/src/front/remoteServer.ts

```diff
@@ -341,6 +321,41 @@ export class RemoteServer implements MsgHandler, ModelSource, SendTarget {
           Utility.wait(1).then(applyPcInv);
           Utility.wait(1.3).then(applyPcInv);
         }
+
+        if(msg.props) {
+          const baseActorValues = new Map<string, unknown>([
+            ["healRate", msg.props.healRate],
+            ["healRateMult", msg.props.healRateMult],
+            ["health", msg.props.health],
+            ["magickaRate", msg.props.magickaRate],
+            ["magickaRateMult", msg.props.magickaRateMult],
+            ["magicka", msg.props.magicka],
+            ["staminaRate", msg.props.staminaRate],
+            ["staminaRateMult", msg.props.staminaRateMult],
+            ["stamina", msg.props.stamina],
+            ["healthPercentage", msg.props.healthPercentage],
+            ["staminaPercentage", msg.props.staminaPercentage],
+            ["magickaPercentage", msg.props.magickaPercentage],
+          ]);
+
+          const player = Game.getPlayer();
```

Comment:
oops, forgot two things:

1. Formatting (RMB -> Format document)
2. Imagine a player has 50% health on the server. The player connects, `damageActorValue` damages his health. The server restarts.  The player reconnects, `damageActorValue` damages his health again killing the player. So please create something like `setActorValuePercentage` that **regenerates if the current percentage is too low**, **damages if the current percentage is too high,** and **does nothing if the current percentage is even with passed one**.